app/src/testDebug/kotlin/org/cru/godtools/ui/tooldetails/ToolDetailsPresenterTest.kt

```diff
@@ -329,6 +331,59 @@ class ToolDetailsPresenterTest {
     }
     // endregion State.drawerState
 
+    // region State.hasTips
+    @Test
+    fun `State - hasTips`() = testScope.runTest {
+        val manifest: Manifest = mockk { every {hasTips} returns true }
+        every { translationsRepository.findLatestTranslationFlow(TOOL, Locale.ENGLISH) }
+            .returns(flowOf(randomTranslation(TOOL, Locale.ENGLISH)))
+
+            coEvery { manifestManager.getManifest(any()) } returns manifest
+
+        createPresenter().test {
+            assertTrue(expectMostRecentItem().hasTips)
+        }
+        }
+    // endregion State.hasTips
+
+    // region State doesn't hasTips
```

Comment:
we use regions to group related functions/properties/code together, so for all 3 of these tests just have them in a single region:

```
// region State.hasTips
fun `State - hasTips`() = ...

fun `State - hasTips - no tips`() = ...

fun `State - hasTips - secondary language has tips`() = ...
// endregion State.hasTips